### ISSUE TITLE
Suggestion: Add "open log file" button to menubar and settings

### ISSUE THREAD

**Issue by Limfjorden** [RimSort/RimSort#436]
**Is your feature request related to a problem? Please describe.**
Currently I have to venture into the file explorer to find the log files. This takes time, and my only other choice is to upload them to the internet.

**Describe the solution you'd like**
It would be easy and convienient to have buttons in RimSort for opening `RimSort.log` and `Rimworld.log`. I think they should be in the "File" menu in the menu bar next to "Upload log". There should also be buttons in the settings menu called "locations" that opens the files.

**Describe alternatives you've considered**

**Additional context**
Tag: Accessibilty

**Comment by Raizanad** [comment]
Call it shortcuts as a drop down menu.
Shortcuts included should be:
SteamCMD
Steam Workshop Mods folders
Local Mods folder
Game mod folder
Root directory folder for the game
Root directory for rimsort
Rimsort user files (as per my other feature request)
Rimworld appdata root folder.

**Comment by oceancabbage** [comment]
Addressing through https://github.com/RimSort/RimSort/pull/493
